src/Agent/NewRelic/Agent/Extensions/NewRelic.Agent.Extensions/Api/IAgentApi.cs

```diff
@@ -179,7 +180,7 @@ public interface IAgentApi
         ///   &lt;body&gt;
         ///   ...
         /// </code></example>
-        string GetBrowserTimingHeader();
```

Comment:
As the result of my comment above, we like to keep 2 overloading methods of the `GetBrowserTimingHeader`.